soc/arm/st_stm32/Kconfig

```diff
@@ -24,6 +24,20 @@ config STM32_ENABLE_DEBUG_SLEEP_STOP
 	  effectivly destroys the use-case of `west attach`. Also
 	  SEGGER RTT and similar technologies need this.
 
+config EXT_FLASH_BASE_ADDRESS
```

Comment:
We need to make this more specific to help users understanding why it's needed
```suggestion
config STM32_MEMMAP_EXT_FLASH_BASE_ADDRESS
```

Reply:
no more used, --> take the `DT_REG_ADDR(DT_INST(0, st_stm32_ospi_nor))`